tests/koji/test_koji.py

```diff
@@ -306,3 +307,103 @@ def test_koji_cache(fake_koji, koji_dir):
 
     # It should have stored something in the cache
     assert cache
+
+
+@patch.dict(
```

Comment:
Could you please add the tests in a new file?  We don't need all the koji tests to go into the same file.

Reply:
Sure, done